AdyenCard/Form/FormCoBadgedCardItemView.swift

```diff
@@ -0,0 +1,113 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import UIKit
+
+/// A view representing a FormCoBadged card Item.
+internal final class FormCoBadgedCardItemView: FormItemView<FormCoBadgedCardItem> {
+
+    private enum Constants {
+        static let viewHeight: CGFloat = 160
+    }
+
+    /// The card brand selection title label item.
+    internal lazy var titleLabel: UILabel = {
+        let titleLabel = UILabel(style: item.style.title)
+        titleLabel.text = item.title
+        titleLabel.accessibilityLabel = ViewIdentifierBuilder.build(
+            scopeInstance: self,
+            postfix: ViewIdentifierBuilder.build(scopeInstance: self, postfix: "cardBadgedCardSelectionTitleLabelItem")
+        )
+        return titleLabel
+    }()
+
+    ///  The card brand selection subtitle label item.
+    internal lazy var subtitleLabel: UILabel = {
+        let subtitleLabel = UILabel(style: item.style.subtitle)
+        subtitleLabel.text = item.subtitle
+        subtitleLabel.numberOfLines = 0
+        subtitleLabel.accessibilityLabel = ViewIdentifierBuilder.build(
+            scopeInstance: self,
+            postfix: ViewIdentifierBuilder.build(scopeInstance: self, postfix: "cardBadgedCardSelectionSubtitleLabelItem")
+        )
+        return subtitleLabel
+    }()
+
+    /// The brand type list item view
+    internal lazy var brandsListView: [SelectableFormItemView] = {
+        item.selectableFormItems.map { item in
+            SelectableFormItemView(item: item)
+        }
+    }()
+
+    /// Initializes the CoBadged card Item view.
+    internal required init(item: FormCoBadgedCardItem) {
+        super.init(item: item)
+        backgroundColor = item.style.backgroundColor
+        addSubview(contentStackView)
+
+        accessibilityIdentifier = item.identifier
+        accessibilityLabel = item.accessibilityLabel
+        isAccessibilityElement = true
+
+        preservesSuperviewLayoutMargins = true
+
+        observe(item.$selectableFormItems) { [weak self] selectableItem in
+            selectableItem.forEach { [weak self] _ in
+                guard let self else { return }
+                removeBrandsListFromView()
```

Comment:
Just a question, not suggestion. Do we need to remove and re-add every time or can we just update the views and only remove/add when the number is 0 or non-zero etc?

Reply:
We are removing and adding only when the change in the card number is detected which results in fetching new binLookup.